redis/src/main/scala/com/avsystem/commons/redis/RedisClusterClient.scala

```diff
@@ -45,6 +45,7 @@ import scala.concurrent.duration._
   * @param config    client configuration - [[ClusterConfig]]
   * @param clusterStateObserver optional observer for monitoring client's state and connections - [[ClusterStateObserver]]
   */
+@deprecated("Redis driver is scheduled for removal. Use a different library, e.g. redisson.", "2.21.0")
```

Comment:
suggestion: Extend this deprecation notice (and other ones as well) with info that this is no longer tested since `2.21.0`. If, for some maintenance reasons, we release new version in the future and someone encounters unexpected behavior, it might be a good clue for searching for the root cause.